types.go

```diff
@@ -14,12 +16,64 @@ type Source struct {
 	Repository string `json:"repository"`
 	RawTag     Tag    `json:"tag,omitempty"`
 
-	Username string `json:"username,omitempty"`
-	Password string `json:"password,omitempty"`
+	Username     string        `json:"username,omitempty"`
+	Password     string        `json:"password,omitempty"`
+	ContentTrust *ContentTrust `json:"content_trust,omitempty"`
 
 	Debug bool `json:"debug,omitempty"`
 }
 
+type ContentTrust struct {
+	Server               string `json:"server"`
+	RepositoryKeyID      string `json:"repository_key_id"`
+	RepositoryKey        string `json:"repository_key"`
+	RepositoryPassphrase string `json:"repository_passphrase"`
+	TLSKey               string `json:"tls_key"`
+	TLSCert              string `json:"tls_cert"`
+}
+
+/* Create notary config directory with following structure
+├── gcr-config.json
+└── trust
+	└── private
+		└── <private-key-id>.key
+└── tls
+	└── <notary-host>
+		├── client.cert
+		└── client.key
+*/
+func (ct *ContentTrust) PrepareConfigDir(src string) (string, error) {
+	configDir := filepath.Join(src, ".notary")
+	os.Mkdir(configDir, os.ModePerm)
```

Comment:
Missing `err` check here and through the rest of the method. There are a few assignments that are unchecked as well.

Reply:
Thanks. :)